### PR TITLE
Use .decrediton instead of .dcrwallet

### PR DESCRIPTION
**Pull request by jcvernaleo**

### CODE REVIEW

**Comment by jcvernaleo** [comment]
Still needs a few more bits so don't review yet.  I just wanted to be able to refer to it in an issue.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,12 @@ dcrd --testnet
 ```
 
 ```bash
-dcrwallet --testnet --experimentalrpclisten=127.0.0.1:19113 --noinitialload --tlscurve=P-256 --onetimetlskey
+dcrwallet --testnet --experimentalrpclisten=127.0.0.1:19113 --noinitialload --tlscurve=P-256 --onetimetlskey --appdata=~/.decrediton
+```
+
+On OSX you should use:
```
Comment (jrick) [inline README.md]: macOS

Comment (jcvernaleo) [inline README.md]: You should just be happy I'm not calling it darwin :)

**Comment by jcvernaleo** [comment]
So this one is ready to go as far as I can tell.

**Pull request merged**